ztp/source-crs/MachineConfigGeneric.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: generic
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    #storage:
+    #  files:
+    #    - contents: ""
```

Comment:
This is interesting, given that we already have an 'extra-manifests' way of adding MachineConfigs at deployment.

Do we know what customers are adding via MachineConfig? This seems dangerous.

Reply:
MachineConfigs definitely have the ability to significantly affect the system. Any use would be at their own risk.